examples/server/server.cpp

```diff
@@ -54,51 +57,93 @@ void bark_print_usage(char **argv, const bark_params &params) {
     fprintf(stderr, "\n");
 }
 
-void bark_params_parse(int argc, char **argv, bark_params &params) {
+void bark_params_parse(int argc, char **argv, bark_params &params, server_params &server_params)
+{
     bool model_req = false;
-    for (int i = 1; i < argc; i++) {
+    for (int i = 1; i < argc; i++)
+    {
```

Comment:
We favour the curly braces on the same line as the statement. Please revert these changes.
```suggestion
    for (int i = 1; i < argc; i++) {
```